guide/getting-started/software.md

```diff
@@ -36,11 +36,11 @@ Illustration sources
 
 # Software overview
 
-The unique history and architecture of Bitcoin has resulted in a few distinct categories of software, with each one serving a distinct purpose. Provided below is an overview of standard and specialized application categories.
+The unique history and architecture of Bitcoin has resulted in several categories of software, with each one serving a distinct purpose. Provided below is an overview of standard and specialized application categories.
 
 ## Command-line interfaces (CLI)
```

Comment:
Will you be re-organizing each section by most used as mentioned in the Google doc?

Reply:
Ah yes. Forgot about that!